### DISCUSSION TITLE
Using neovim as the text editor of browser text inputs

### DISCUSSION THREAD

**Discussion by mostafatouny** [qutebrowser/qutebrowser#8374] [Ideas] [answered]
Hello,
I found and tested [Ghost Text](https://github.com/fregante/GhostText) which allows users to use their favorite text editor, including vim and neovim, to write on text inputs of the browser. I believe integrating it into qutebrowser aligns with its vision and its user base.

**Comment by bitraid** [comment]
qutebrowser has this functionality for over 10 years. There is the [cmd-edit](https://qutebrowser.org/doc/help/commands.html#cmd-edit) command, with default bind to `<Ctrl-e>`. You can configure it with the [editor.command](https://qutebrowser.org/doc/help/settings.html#editor.command) setting. See https://github.com/qutebrowser/qutebrowser/issues/5340 for examples.

**Reply by mostafatouny** [reply]
If I understand, `cmd-edit` modifies the qutebrowser command. But we can't use to modify text inputs of a page like:

![](https://user-images.githubusercontent.com/1402241/226180991-5be2517c-ef2a-4884-8427-f4080ff0eac8.gif)

**Reply by bitraid** [reply]
Your understanding is wrong.

**Reply by mostafatouny** [reply]
Thank you for your time. I shall spend more time with the documentation.

**Comment by mostafatouny** [comment, marked as answer]
Following @bitraid advise to check the documentation I found [edit-text](https://qutebrowser.org/doc/help/commands.html#edit-text) which allows to modify text inputs with external text editors. In my `config.py` file, I set `c.editor.command = ["kitty", "--single-instance", "-T", "auxiliary text edit", "nvim", "{file}", "+startinsert", "+call cursor({line}, {column})"]`, following the examples given by @bitraid also. Thank you

**Reply by bitraid** [reply]
Well done! I just realized that I gave you the `edit-cmd` by mistake, but you managed to figure it out :+1:
